dev-python/pybedtools/pybedtools-0.11.0.ebuild

```diff
@@ -24,11 +24,9 @@ RDEPEND="
 	dev-python/pandas[${PYTHON_USEDEP}]
 	dev-python/matplotlib[${PYTHON_USEDEP}]
 "
+BDEPEND="dev-python/cython[${PYTHON_USEDEP}]"
```

Comment:
dependency on cyhthon is not required when `DISTUTILS_EXT=1` is set

Reply:
fixed

Reply:
I've removed this and am now getting a QA Notice:

> * QA Notice: Package seems to be missing a BDEPEND on dev-python/cython.

Reply:
Then please check if you are using the most recent version of the eclass.

Reply:
I have the main tree synced at `2025-01-27 11:18:44 UTC`, so this should include the latest `distutils-r1` eclass. I still get the QA notice